static/app/components/feedback/useFeedbackListQueryParams.tsx

```diff
@@ -0,0 +1,41 @@
+import {Location} from 'history';
+
+import {FeedbackListQueryParams} from 'sentry/utils/feedback/types';
+import {decodeList, decodeScalar} from 'sentry/utils/queryString';
+
+export default function useFeedbackListQueryParams({
```

Comment:
This seems like it could be used in other places too

Reply:
yeah, it's a portion of ViewEvent in a way.

I was prototyping not using ViewEvent here, and michelle is also bringing that into the dead/rage click tables, So we'll have two examples to riff on.